README.md

```diff
@@ -8,18 +8,18 @@ Docmaker is a **WIP** markdown documentation rendering tool.
 
 `$ docmaker <dir>`
 
-Docmaker will try to find `docmaker.yaml` in the current directory or above and resolve all filepaths specified in the config relative to this `project directory`.
+Docmaker will try to find `docmaker.yml` in the current directory or above and resolve all filepaths specified in the config relative to this `project directory`.
```

Comment:
I figured going with `.yml` was better since most YML format files use that extension:
- `.gitlab-ci.yml` doesn't accept `.yaml`
- `docker-compose.yml` is the default path according to their documentation